apps/charterafrica/src/lib/openAfrica.js

```diff
@@ -72,17 +72,19 @@ function formatDatasets(datasets) {
       name,
       notes,
       url: url?.trim(),
+      href: `https://openafrica.net/${type}/${id}`,
+      localHref: `${path}/${id}`,
```

Comment:
`href` should be to charter.africa page. This is what the card should open when clicked. You can use `url` and `source` to mean either openAFRICA link or link to where openAFRICA got the dataset from i.e. source of the data.